crates/aptos/src/node/mod.rs

```diff
@@ -1074,6 +1074,12 @@ pub struct RunLocalTestnet {
     #[clap(long)]
     do_not_delegate: bool,
 
+    /// If enabled, will execute transactions with the gas profiler.
+    ///
+    /// The result will be saved to the `gas-profiling` directory as a flamegraph.
+    #[clap(long)]
+    enable_gas_profiling: bool,
```

Comment:
@gregnazario Even though it's possible to enable gas profiling by modifying the node config, I feel like that would still be too inconvenient for a user so I added this command line option. Feel free to let me know if you think it should be done differently.